contracts/src/interfaces/IHyperdriveCore.sol

```diff
@@ -190,27 +206,6 @@ interface IHyperdriveCore is IMultiTokenCore {
     /// @param _status True to pause all deposits and false to unpause them.
     function pause(bool _status) external;
 
-    /// @notice Allows governance to transfer the fee collector role.
-    /// @param _who The new fee collector address.
-    function setFeeCollector(address _who) external;
```

Comment:
Is the idea that we just use the Factory's, for example, `updateFeeCollector` function to set the fee collector now?